File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        HashSet<Integer> set=new HashSet<>();
        for(int n:nums){
            set.add(n);
        }
        int ls=0;
        for(int n:set){
            if(!set.contains(n-1)){
            int cn=n;
            int cs=1;
            while(set.contains(cn+1)){
                cn++;
                cs++;
            }
            ls=Math.max(ls,cs);
        }
    }
    return ls;
    }
}

File: Data Structures & Algorithms/find-pivot-index/submission-0.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        n=len(nums)
        for i in range(n):
            ls=rs=0
            for l in range(i):
                ls+=nums[l]
            for r in range(i+1,n):
                rs+=nums[r]
            if ls==rs:
                return i
        return -1